docs/snippets/react/your-component-with-decorator.js.mdx

```diff
@@ -1,14 +1,16 @@
 ```js
 // YourComponent.stories.js | YourComponent.stories.jsx
 
+import YourComponent from './YourComponent';
```

Comment:
```suggestion
import { YourComponent } from './YourComponent';
```

Reply:
I'll leave it as it is if you don't mind, as per the comment above.